src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -432,6 +436,14 @@ define([
                     }.bind(this));
                 }
 
+                this.$items[id].on('mouseenter', function() {
+                    this.startDownloadDropdown(id);
+                }.bind(this));
+
+                this.$items[id].on('mouseleave', function() {
+                    this.stopDownloadDropdown(id);
```

Comment:
Haven't you build some logic to hide all other opened dropdowns when clicking on another one? Shouldn't this be removed with that change?